docs/content/en/docs/crd-ref/metrics/v1alpha3/_index.md

```diff
@@ -191,6 +191,23 @@ _Appears in:_
 | `query` _string_ | Query represents the query to be run. It can include placeholders that are defined using the go template syntax. More info on go templating - https://pkg.go.dev/text/template |
 
 
+#### IntervalResult
+
+
+
+
+
+_Appears in:_
+- [KeptnMetricStatus](#keptnmetricstatus)
+
+| Field | Description |
+| --- | --- |
+| `value` _string_ | Value represents the resulting value |
```

Comment:
these docs seem to be out of sync with what's in the code, so I think they need to be regenerated

Reply:
I ran `./.github/scripts/generate-crd-docs/generate-crd-docs.sh` again but the result is same.

Reply:
found the issue now - there was one comment that needed to be adapted in the metric type - see the suggestion in https://github.com/keptn/lifecycle-toolkit/pull/1900/files#r1326870168

Reply:
Thanks